.github/workflows/build-binaries.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - "releases/*"
+  pull_request:
```

Comment:
Can we remove this?

Reply:
Yes, sorry, will do in a subsequent PR since this is almost passing CI.

Reply:
Done